app/template/[templateId]/page.tsx

```diff
@@ -120,49 +223,190 @@ const TemplateEditPage: React.FC = () => {
                     id={question.id}
                     name={question.name}
                     link={question.link}
-
-
                   />
                 ))}
+                <AddQuestionButton
+                  href={`/template/${template.id}/q/new?section_id=${section.id}`}
+                />
               </div>
+
             ))}
-            <Button
-              className="add-question-button tertiary mx-auto">
-              + Add question
-            </Button>
 
           </div>
 
-          <div className="section-add-cont">
-            <Button
-              className="add-section-button tertiary"
-            >
-              + Add new section
-            </Button>
+          <AddSectionButton href={`/template/${template.id}/section/new`}/>
+
+
+        </div>
+        <aside className="sidebar">
+          <div className="sidebar-inner">
+            <h2>Status</h2>
+            <div className="sidebar-section">
+
+              <Button data-secondary className="my-3 secondary"
+                      onPress={() => console.log('Save draft')}>
+                Save as draft
+              </Button>
+
+              <Button data-tertiary className="my-3"
+                      onPress={() => console.log('Preview')}>Preview
+                template</Button>
+
+            </div>
+
+            <div className="sidebar-section">
+              <h5 className="sidebar-section-title">Published Status</h5>
+              <div className="status">
+                <p>
+                  Draft <Link href='#'>Edit</Link>
+                </p>
+              </div>
+            </div>
+
+            <div className="sidebar-section">
+              <h5 className="sidebar-section-title">Visibility Settings</h5>
+              <div className="status">
+                <p>
+                  Not Published <Link href='#'>Edit</Link>
+                </p>
+              </div>
+            </div>
+
+
+            <div className="sidebar-section">
+              <h5 className="sidebar-section-title">Feedback &
+                Collaboration</h5>
+              <div className="description">
+                <p>
+                  Allow people to access, edit or comment on this plan
+                </p>
+                <p>
+                  <Link className="learn-more"
+                        href="/template/tpl_abcdef123456/access">
+                    Manage Access
+                  </Link>
+                </p>
+              </div>
+            </div>
+
+
+            <div className="sidebar-section">
+              <Button
+                className="my-3"
+                onPress={() => setPublishModalOpen(true)}
+              >
+                Publish template
```

Comment:
When you click on `Publish template` the modal opens but is not scrolled to the top.

Reply:
Maybe Im not understanding what you mean here - do you think the page should scroll to the top? I'd prefer where possible to "keep" the scroll position

Reply:
I meant that the modal opens, but I can't see the top of the modal.